modules/social_features/social_group/modules/social_group_flexible_group/social_group_flexible_group.install

```diff
@@ -640,13 +640,22 @@ function social_group_flexible_group_update_11901(): string {
   return $updater->logger()->output();
 }
 
+/**
+ * Disable revisions for flexible group.
+ */
+function social_group_flexible_group_update_111102(): void {
```

Comment:
should this also get a new number?

Reply:
This one has a milestone of 11.10.1
https://github.com/goalgorilla/open_social/pull/3517

Thats why I kept it as this same number.